src/components/DefinitionList/DefinitionList.tsx

```diff
@@ -0,0 +1,163 @@
+            );
+        }
+
+        if (typeof note === 'object') {
+            noteElement = <HelpPopover className={popoverClassName} {...note} />;
+        }
+    }
+    return noteElement;
+}
+
+export function DefinitionList({
+    items,
+    responsive,
+    keyMaxWidth,
+    valueMaxWidth = 'auto',
+    className,
+    itemClassName,
+}: DefinitionListProps) {
+    const keyStyle = keyMaxWidth
+        ? {
+              flexBasis: keyMaxWidth,
+          }
+        : {};
+
+    const valueStyle =
+        typeof valueMaxWidth === 'number'
+            ? {
+                  flexBasis: valueMaxWidth,
+                  maxWidth: valueMaxWidth,
+              }
+            : {};
+    return (
+        <dl className={b({responsive}, className)} role="list">
+            {items.map(
+                ({
+                    name,
+                    href,
+                    content,
+                    title,
+                    copyText,
+                    note,
+                    key,
+                    copyIconOver,
+                    multilineName,
+                }) => {
+                    const term = href ? <Link href={href}>{name}</Link> : name;
+                    const definitionContent = content ?? '—';
+                    const definition = copyText ? (
+                        <div className={b('copy-container', {'icon-inside': copyIconOver})}>
+                            <span>{definitionContent}</span>
+                            <ClipboardButton
+                                size="s"
+                                text={copyText}
+                                className={b('copy-button')}
+                            />
+                        </div>
+                    ) : (
+                        definitionContent
+                    );
+                    const noteElement = (
+                        <React.Fragment>
+                            &nbsp;
+                            {getNoteElement(note)}
+                        </React.Fragment>
+                    );
+                    return (
+                        <div key={key ?? name} className={b('item', itemClassName)} role="listitem">
+                            <div
+                                className={b('term-container', {multiline: multilineName})}
+                                style={keyStyle}
+                            >
+                                <div className={b('term-wrapper')}>
+                                    <dt className={b('term')} title={name} role="term">
+                                        {term}
+                                    </dt>
+                                    {multilineName && noteElement}
+                                </div>
+                                {!multilineName && noteElement}
+                                <div className={b('dots', {'with-note': Boolean(note)})} />
+                            </div>
+                            <dd
```

Comment:
Correct structure will be:

dl
-- div
---- dt
---- dd
-- div
---- dt
---- dd